### ISSUE TITLE
Outdated "Roadmap" page ?

### ISSUE THREAD

**Issue by fabm22** [dojo/meta#217]
Hi, sorry to open an issue for this, but the [roadmap page](https://github.com/dojo/meta/wiki/Roadmap) seems now notably outdated.

And, in general, I m trying to have an idea to when to expect a "near" Release Candidate level of the development of Dojo 2 ??

Thanks for answer.

**Comment by agubler** [comment]
Hey @fabm22. Yes apologies, the roadmap is very out of date now. Hopefully we should be able to get that updated this week to reflect the current timelines and deliverables of `beta5` and the first Release Candidate.

Currently the last Beta (beta5) is planned for around mid December, with a release candidate early January 2018. Once the exact dates are confirmed we'll make sure that they are added to the roadmap for visibility.

Thank you for taking the time to open an issue and for the interest in Dojo 2 😄

**Comment by kitsonk** [comment]
Yeah, we have had it on our backlog in our internal planning for a while.  Apologies for not keeping it updated.
